bootstrap/kind-config.yml

```diff
@@ -7,4 +7,10 @@ nodes:
         hostPort: 30000
         listenAddress: "0.0.0.0"
         protocol: tcp
+    image: kindest/node:v1.20.7@sha256:cbeaf907fc78ac97ce7b625e4bf0de16e3ea725daf6b04f930bd14c67c671ff9
   - role: worker
```

Comment:
Additional worker nodes were added based on the documentation for the Operator.  After a successful execution of the pipeline the nodes were reduce to 2, but the output showed this configuration to be invalid and did not complete.  The additional nodes has not increase the execution time for the pipeline too much.